src/status_im/data_store/realm/accounts.cljs

```diff
@@ -1,19 +1,20 @@
 (ns status-im.data-store.realm.accounts
   (:require [status-im.data-store.realm.core :as realm]))
 
-(defn get-all []
-  (realm/get-all realm/base-realm :account))
-
 (defn get-all-as-list []
-  (->> (get-all)
+  (->> (realm/get-all realm/base-realm :account)
        realm/js-object->clj
+       (map #(update % :settings realm/deserialize))
        (mapv #(realm/fix-map % :networks :id))))
 
+
 (defn get-by-address [address]
-  (realm/fix-map (realm/get-one-by-field-clj realm/base-realm :account :address address)
-                 :networks :id))
+  (-> (realm/get-one-by-field-clj realm/base-realm :account :address address)
+      (update :settings realm/deserialize)
+      (realm/fix-map :networks :id)))
 
 (defn save [account update?]
   (realm/write realm/base-realm
-               (let [account' (realm/fix-map->vec account :networks)]
+               (let [account' (realm/fix-map->vec account :networks)
+                     account' (update account' :settings realm/serialize)]
```

Comment:
It should either `account''` ( 😄 ) or you can extend previous form with threading macro. Otherwise this shadowing looks even worse then `symbol'''''''''` thing imo

Reply:
Who doesn't like `'` ! 
Threading sounds good.